cob_twist_controller/include/cob_twist_controller/cob_twist_controller_data_types.h

```diff
@@ -78,6 +83,18 @@ struct JointStates
     KDL::JntArray last_q_dot_;
 };
 
+struct ActiveCartesianDimension {
```

Comment:
It is some kind of enable/disable Struct indeed...
...but the elements are used within a double multiplication...
(so I haven't yet decided on a best name yet...)